README.md

```diff
@@ -1,76 +1,132 @@
-* [Bunyan 1+](https://github.com/trentm/node-bunyan) - `use('bunyan')`
-* [cassandra-driver](https://github.com/datastax/nodejs-driver) - `use('cassandra-driver')`
-* [DNS](https://nodejs.org/api/dns.html) - `use('dns')`
-* [elasticsearch 10+](https://www.elastic.co/guide/en/elasticsearch/client/javascript-api/current/index.html) - `use('elasticsearch')`
-* [Express 4+](http://expressjs.com/) - `use('express')`
-* [GraphQL 0.10+](https://github.com/graphql/graphql-js) - `use('graphql')`
-* [hapi 2+](https://hapijs.com/) - `use('hapi')`
-* [http/https](https://nodejs.org/api/http.html) - `use('http')`, `use('https')`
-* [ioredis 2+](https://github.com/luin/ioredis) - `use('ioredis')`
-* [Koa 2+](https://koajs.com/) - `use('koa')`
-* [Knex 0.10+](https://knexjs.org/) - `use('bluebird'); use('knex')` (depends on bluebird instrumentation)
-* [Memcached 2.2+](https://github.com/3rd-Eden/memcached) - `use('memcached')`
-* [MongoDB-Core 2+](https://github.com/mongodb-js/mongodb-core) - `use('mongodb-core')`
-* [mysql 2+](https://github.com/mysqljs/mysql) - `use('mysql')`
-* [MySQL2 1+](https://github.com/sidorares/node-mysql2) - `use('mysql2')`
-* [Nest](https://nestjs.com/) - `use('nest')`
-* [Net](https://nodejs.org/api/net.html) - `use('net')`
-* [node-postgres 4+](https://github.com/brianc/node-postgres) - `use('pg')`
-* [Pino 2+](http://getpino.io/#/) - `use('pino')`
-* [Q 1+](https://github.com/kriskowal/q) - `use('q')`
-* [Redis 0.12+](https://github.com/NodeRedis/node_redis) - `use('redis')`
-* [restify 3+](http://restify.com/) - `use('restify')`
-* [Sails 1+](https://sailsjs.com) - `use('sails')`
-* [Socket.IO 1.2+](https://socket.io) - `use('socket.io')`
-* [when.js 3+](https://github.com/cujojs/when) - `use('when')`
-* [winston 1+](https://github.com/winstonjs/winston) - `use('winston')`
-
-#### About
-The SignalFx-Tracing Library for JavaScript is a fork of the DataDog APM JavaScript Tracer that has been modified to provide Zipkin v2 JSON formatting, B3 trace propagation functionality, and properly annotated trace data for handling by [SignalFx Microservices APM](https://docs.signalfx.com/en/latest/apm/apm-overview/index.html).
+The SignalFx Tracing Library for JavaScript automatically instruments your
+Node.js application to capture and report distributed traces to SignalFx
+with an OpenTracing-compatible tracer.
+
+The tracer has constant sampling (i.e., 100% chance of tracing) and
+reports every span. Where applicable, context propagation uses
+[B3 headers](https://github.com/openzipkin/b3-propagation).
+
+For more information about configuring and using the agent, see the
+[examples](https://github.com/signalfx/tracing-examples/tree/master/signalfx-tracing/signalfx-nodejs-tracing).
+
+For advanced configuration information, see
+[SignalFx Tracing Library for JavaScript - API](./docs/API.md).
+
+## Requirements and supported software
+
+The library supports Node.js versions 4.7+, 6.9+, and 8+.
```

Comment:
I'm not sure if this is true anymore. I think we only support 8+ now. @rmfitzpatrick can you confirm?

Reply:
Our current version should support all of these, but it will change with the next major update.